src/services/general.api.js

```diff
@@ -2,6 +2,7 @@ import { createApi } from "@reduxjs/toolkit/query/react";
 import { baseQuery } from "config/baseQuery";
 
 export const generalApi = createApi({
+  refetchOnFocus: true,
```

Comment:
Pourquoi refetchOnFocus ?

Reply:
meme chose que la remarque précédente , le metier a demandé que si une mission est ajouter par un call api,ou par un autre user ,  je dois la voir ....., example je suis sur l'onglet missions, il y a 10 missions, un autre utilisateur ajoute une mission, moi si je change d'onglet , je doit voir 11 missions, après niveau pérformance ça va faire un call a chaque focus , mais ça a été exigé , si tu vois une autre manière de le faire je suis preneur

Reply:
il suffira de faire un appel après le submit d'une mission de `getAllMissions `et ça sera automatiquement remis a jour, parce que de cette maniére a chaque focus in sur la fenetre toutes les requetes se rexecuterons